.travis.yml

```diff
@@ -8,3 +8,5 @@ go:
 
 script:
  - go test -short ./...
+ - go tool vet .
```

Comment:
Other Prometheus repos (via Makefile.common, see next comment) use `staticcheck`. What do you think about that? We needed a `go get -u honnef.co/go/tools/cmd/staticcheck` first, but that would bring the check level in line.

Reply:
That sounds a good idea. But for now `staticcheck ./...` fails with

```
prometheus/examples_test.go:166:14: prometheus.UninstrumentedHandler is deprecated: Use promhttp.Handler instead. See there for further documentation.  (SA1019)
prometheus/examples_test.go:198:26: prometheus.Handler is deprecated: Please note the issues described in the doc comment of InstrumentHandler. You might want to consider using promhttp.InstrumentedHandler instead.  (SA1019)
```
I am too new to `prometheus/client_golang` to come with a proper fix for this 2 warnings. So not sure what to do w.r.t this PR.